libkbfs/folder_branch_ops.go

```diff
@@ -2287,14 +2287,6 @@ func (fbo *folderBranchOps) getConvID(
 
 func (fbo *folderBranchOps) sendEditNotifications(
 	ctx context.Context, rmd ImmutableRootMetadata, body string) error {
-	// For now only write out the notifications if we're in test mode,
-	// just in case we decide to change the notification format before
-	// we launch.  TODO: turn this on for admins once we can test it
-	// on staging.
```

Comment:
Do we still want the "admins" gate?

Reply:
Now that I have an end-to-end kbfsdocker test working, I'm ok enabling it for non-admins too.